cypress-tests/cypress/e2e/admin/pageBuilder/blocks/pageBlockPreview.cy.ts

```diff
@@ -0,0 +1,72 @@
+import { generateAlphaLowerCaseId } from "@webiny/utils/generateId";
+context("Page Builder - Blocks", () => {
+    beforeEach(() => {
+        cy.login();
+        cy.pbDeleteAllBlocks();
+        cy.pbDeleteAllBlockCategories();
+        cy.pbDeleteAllTemplates();
+    });
+    const blockName = ["Block1Name"];
+    const headerTitle = "Header test";
+    const headerTitleUpdate = "Edited Header Text";
+    const templateData = "tester";
+
+    const blockCategoryData = {
+        name: generateAlphaLowerCaseId(10),
+        slug: generateAlphaLowerCaseId(10),
+        icon: "icon-name",
+        description: generateAlphaLowerCaseId(10)
+    };
+
+    it("Should be able to programatically create and assert a block exists with a proper heading", () => {
+        cy.pbCreateCategoryAndBlocks({
+            blockCategory: blockCategoryData,
+            blockNames: blockName,
+            content: {
+                type: "heading",
+                text: headerTitle
+            }
+        });
+
+        cy.visit("/page-builder/page-blocks");
+        // Asserts the programatically created block contains the correct data.
+        cy.contains(blockCategoryData.name).click();
```

Comment:
Can we target the container in which we expect this? Like we did it in the past:

```ts
      cy.findByTestId("default-data-list").within(() => {
            cy.get("li")
                .first()
                .within(() => cy.findByText(titleString4).should("exist"));
        });
```

Just so we're not doing a select across the whole document. More importantly, if somebody adds the same string in the future in the same view, this test would start failing (it would find two occurences). I want to basically minimize future unneeded failures of tests as much as possible, because it's Cypress and, as we all know, it's both flaky and slow. 🙃